package.json

```diff
@@ -9,6 +9,7 @@
     "version": "independent"
   },
   "scripts": {
+    "local-ci": "yarn clean && yarn && yarn lint && yarn build && yarn type-check && yarn test",
```

Comment:
Do we want to link-packages here too, since PRs may be working across packages?

Could be `yarn clean-link && yarn lint && yarn type-check && yarn test`

Reply:
good thought but not quite - `yarn link-packages` in `lib` is just setting the packages up so they can be consumed by `web`.

`yarn clean` breaks all the links, and reflects the state the CI is in locally, so we don't need to `clean-link` here